libraries/src/application/Helper.php

```diff
@@ -7,14 +7,22 @@
  * @license     GNU General Public License version 2 or later; see LICENSE
  */
 
+namespace Joomla\Cms\Application;
+
 defined('JPATH_PLATFORM') or die;
 
+use \JFactory;
+use \JLog;
+use \JFilterOutput;
+use \JInstaller;
+use \JText;
+
 /**
  * Application helper functions
  *
  * @since  1.5
  */
-class JApplicationHelper
+class Helper
```

Comment:
I think consistency is the primary goal of namespaces. You should have things ending in Helper or Site. Take a look at things like Guzzle and Amazon AWS SDK for PHP. They're not at all shy on following this approach.